installation/single_machine_rhel.rst

```diff
@@ -32,28 +32,28 @@ Let's create directories for those nodes to store their data. For convenience in
   export PATH=$PATH:/usr/pgsql-9.6/bin
 
   cd ~
-  mkdir -p citus/master citus/worker1 citus/worker2
+  mkdir -p citus/coord citus/worker1 citus/worker2
```

Comment:
Two quick questions.

1. Do we want to change this or leave it as-is?
2. If we change it, what do you think about "coord" --> "coordinator"?